Classes/Controller/ListViewController.php

```diff
@@ -87,6 +87,8 @@ public function mainAction()
         $GLOBALS['TSFE']->fe_user->setKey('ses', 'widgetPage', $widgetPage);
 
         // get all sortable metadata records
+        // TODO: Sebastian fragen
+        // TODO: Include also subentries if available.
```

Comment:
Are these TODOs still relevant? The two-level metadata feature should be implemented completely. So either these TODOs are done (and should be removed) or should be done.